api_spec.yaml

```diff
@@ -3246,9 +3246,496 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
-          
-      
+  
```

Comment:
remove statusTenantDoesNotExistError and replace it with returning a 400 status code (bad request)

Reply:
Done